SQL/0000-00-00-schema.sql

```diff
@@ -2364,4 +2364,27 @@ CREATE TABLE `user_psc_rel` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 INSERT INTO user_psc_rel (UserID, CenterID) SELECT 1, CenterID FROM psc;
 
+DROP TABLE IF EXISTS `monitoring`;
+CREATE TABLE `monitoring` (
+  `CandID` int(6) NOT NULL,
+  `visit_label` varchar(255) DEFAULT NULL,
+  `monitored` enum('Y','N') NOT NULL DEFAULT 'N',
+  `date_monitored` date DEFAULT NULL,
+  `monitor_id` int(10) unsigned DEFAULT NULL,
+  PRIMARY KEY (`CandID`,`Visit_label`),
+  CONSTRAINT `FK_monitoring_1` FOREIGN KEY (`monitor_id`) REFERENCES `examiners` (`examinerID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+DROP TABLE IF EXISTS `monitoring_history`;
+CREATE TABLE `monitoring_history` (
+  `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `CandID` int(6) NOT NULL,
+  `visit_label` varchar(255) DEFAULT NULL,
+  `flag` enum('Y','N') NOT NULL DEFAULT 'N',
+  `monitored` enum('Y','N') NOT NULL DEFAULT 'N',
+  `date_monitored` date DEFAULT NULL,
+  `monitor_id` int(10) unsigned DEFAULT NULL,
+  `data_entry_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `entry_staff` varchar(255) DEFAULT NULL,
```

Comment:
Shouldn't we use userID instead of entry_staff to be consistent with other tables.